pkg/ddevapp/config.go

```diff
@@ -712,6 +711,21 @@ func (app *DdevApp) FixObsolete() {
 			}
 		}
 	}
+
+	// Remove old .global_commands directory
+	legacyCommandDir := app.GetConfigPath(".global_commands")
+	if fileutil.IsDirectory(legacyCommandDir) {
```

Comment:
Why not just `os.RemoveAll(legacyCommandDir)` instead of this whole stanza?

Reply:
Because I didn't know that existed 😂

Reply:
Done